theseus_gui/src/components/ui/Instance.vue

```diff
@@ -39,7 +116,7 @@ const props = defineProps({
   justify-content: center;
   cursor: pointer;
   padding: 0.75rem;
-  transition: 0.1s ease-in-out all;
+  z-index: 40;
```

Comment:
don't use random z indexes in general, this goes over the authentication wrapper for example. why does this need a custom z index?